src/focus_manager/txm_focus_manager.js

```diff
@@ -263,8 +263,11 @@ export class TXMFocusManager {
             const now = Date.now();
             const elapsedTime = now - this._lastKeyEventTimestamp;
             if (keyCode == this._lastKeyCode && elapsedTime <= throttleDelay) {
-                // Swallow the excess key event.
-                return true; // handled
+                // Swallow the excess key event and prevent it from propagating to parent's focusManager
+                event.preventDefault();
```

Comment:
See about leveraging the existing logic doing this at the bottom of this function.  One idea is to make use of the `handled` variable.